src/index.ts

```diff
@@ -69,7 +69,7 @@ export type Parser = (rule: Rule, ctx: Context) => Parser;
 export const runSass = function (
   trueOptions: TrueOptions,
   src: string,
-  sassOptions?: Options<'sync'> | StringOptions<'sync'>,
+  sassOptions?: any,
```

Comment:
This is unfortunate, but unavoidable if we really want to support any Sass implementation.

Reply:
I haven't tried it, but could you do something like `Options<'sync'> | StringOptions<'sync'> | any`? Not sure if that just removes any benefit anyways, though.